_overviews/plugins/index.md

```diff
@@ -42,18 +42,17 @@ include:
 - Alternate compiler back ends such as [Scala.js](https://www.scala-js.org), [Scala Native](http://scala-native.org), and
   [Fortify SCA for Scala](https://developer.lightbend.com/docs/fortify/current/).
 - Linters such as [Wartremover](https://www.wartremover.org) and [Scapegoat](https://github.com/sksamuel/scapegoat).
-- Plugins that support reformatting and other changes
-  to source code, such as [scalafix](https://scalacenter.github.io/scalafix/) and [scalafmt](https://scalameta.org/scalafmt/) (which are
-  built on the [semanticdb](https://scalameta.org/docs/semanticdb/guide.html) and [scalahost](https://github.com/Duhemm/scalahost) compiler plugins).
```

Comment:
did you mean to remove these lines?

Reply:
Yeah, see this thread: https://github.com/scala/docs.scala-lang/pull/2011#discussion_r630107019

Reply:
actually, the commit explains more too: https://github.com/scala/docs.scala-lang/pull/2011/commits/9aaff97197d191266ceb479c4273ca6b06b86492

>  scalafix/scalafmt aren't compiler plugins, scalahost isn't maintained.